$OpenBSD: patch-src_GuiEvents_cpp,v 1.1 2012/04/10 13:08:37 dcoppa Exp $

Save met file when a download is paused or stopped
(upstream git commit d9b4458b1ea214af48bea97cd8482016a0eccbb9)

--- src/GuiEvents.cpp.orig	Mon Jun 13 10:50:25 2011
+++ src/GuiEvents.cpp	Tue Apr 10 14:35:37 2012
@@ -647,6 +647,7 @@ namespace MuleNotify
 	void PartFile_Pause(CPartFile* file)
 	{
 		file->PauseFile();
+		file->SavePartFile();
 	}
 
 	void PartFile_Resume(CPartFile* file)
@@ -658,6 +659,7 @@ namespace MuleNotify
 	void PartFile_Stop(CPartFile* file)
 	{
 		file->StopFile();
+		file->SavePartFile();
 	}
 
 	void PartFile_PrioAuto(CPartFile* file, bool val)
